engine/index.js

```diff
@@ -381,6 +389,7 @@ function saveData(features, dbTestNumber) {
         return redis.hmset(client, 'changelog', date, JSON.stringify(changedData));
       }
       console.log('DEBUG: no changes found');
+      return Promise.resolve();
```

Comment:
Which is completely useless here :(
Disable the related warning in .eslintrc or keep this change, you decide.